docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -1,12 +1,11 @@
 +++
 aliases = ["/docs/grafana/latest/administration/configuration/", "/docs/grafana/latest/installation/configuration/"]
 description = "Configuration documentation"
-keywords = ["grafana", "configuration", "documentation"]
```

Comment:
missing alias for old location here, very important as this is highly linked to page

Reply:
The current location of this file is: docs/grafana/latest/administration/configuration/, which I've got documented as an alias, so I think we're ok here.